### PR TITLE
Bump to v1.4.0

### PR DESCRIPTION
**Pull request by netoax**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This PR bumps the current version to v1.4.0. The main changes are:

- Update the stack `yaml` files with the newer version of the KNoT services.
- Fix the `traeffik` version to be v1.7 since the latest (v2.x) isn't compatible with the current configuration.

Does this close any currently open issues?
------------------------------------------
Nops.

Any relevant logs, error output, etc?
-------------------------------------
Nops.

Any other comments?
-------------------
Nops.

### CODE REVIEW

File: stacks/dev/stage-1.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ services:
           memory: 300M
   # Proxy
   traefik:
-    image: traefik
+    image: traefik:v1.7
```
Comment (ramonhpr) [inline stacks/dev/stage-1.yml:71]: This change doesn't seem to be related to the commit bump. 
Why do you need to fix the traefik version?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+  - Get data
+  - Publish data
+  - Listen data
+  - Update schema
+  - Create session token
+  - Delete device
+
+### Bug Fixes
+
+- Update packages to fix vulnerability issues
```
Comment (ramonhpr) [inline CHANGELOG.md:19]: Update the traefik bug fix

**Review by ramonhpr — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (netoax) [reply CHANGELOG.md:19]: Done.

File: stacks/dev/stage-1.yml (hunk 1 above)
Comment (netoax) [reply stacks/dev/stage-1.yml:71]: I've created another commit related to this change: _51dd37a1267d517e13f04169ce7d6c7492cd3ccd_ .

**Review by lcbm — APPROVED** [review]

**Review by ramonhpr — APPROVED** [review]
